HelpHunterBE/Search.cs

```diff
@@ -0,0 +1,14 @@
+public class SearchCriteria
+{
+    public string Location { get; set; }
+    public decimal PriceMax { get; set; }
+    public decimal PriceMin { get; set; }
+    public string CategoryOrServiceName { get; set; }
+    public decimal RatingMax { get; set; }
+    public decimal RatingMin { get; set; }
```

Comment:
Rating mamy 1,2,3,4,5 czy 1.5,2,2.5 itd? Bo jak 1,2,3,4,5 to imo int

Reply:
liczymy później średnią z np. 4, 2, 1 i 3 więc może wyjść po przecinku.

Reply:
oke, w takim razie decimal będzie spoko (lepszy pewnie float, ale to drobnostka, nie ma co zmieniać)